SchedulerJobs/SchedulerJobs.Services/VideoApi/Contracts/ClosedConferencesResponse.cs

```diff
@@ -0,0 +1,12 @@
+using System;
```

Comment:
And the name of this one as well. Sounds the same as ExpiredConferencesResponse

Reply:
renamed to "ClosedConferenceWithImHistoryResponse"